.github/workflows/release-please.yml

```diff
@@ -17,50 +20,21 @@ jobs:
         with:
           command: manifest
           # In case we need to add a PAT and have the pr's trigger actions
-          #token: ${{ secrets.PAT }}
+          # token: ${{ secrets.PAT }}
           # See how to configure file https://github.com/googleapis/release-please/blob/main/docs/manifest-releaser.md
           # See output info at https://github.com/google-github-actions/release-please-action#configuration for variables
           # Trigger more steps using "if: ${{ steps.release.outputs.release_created }}"
-
   build:
+    name: Build
     needs: release-please
-    name: "Build"
-
-    runs-on: ${{ matrix.os }}
-
-    if: ${{ needs.release-please.outputs.release_created }}
-
-    strategy:
-      matrix:
-        include:
-          - os: ubuntu-latest
-            name: linux
-            yarn-task: build:linux
-          - os: macos-latest
-            name: mac
-            yarn-task: build:mac
-          - os: windows-latest
-            name: win
-            yarn-task: build:win
-
+    uses: ./.github/workflows/build.yml
+  publish:
+    name: Pubish
```

Comment:
It looks like a minor typo error :)

Reply:
Sorry, my mistake, it should be "publish"